### ISSUE TITLE
Short Question to clear Screen

### ISSUE THREAD

**Issue by cyberpunkbln** [TimonPost/crossterm#77]
Hello,

i'm a relative new in rust and i'm hanging here at one simple solution to clear the terminal. Your solution use the winapi pure, or? No ansi escape sequence.

I have searched the whole internet:), and not find an solution why a cargo run accept ansi escapes codes and directly run of an .exe don't display the codes:).

Is it possible to easily strip down your crate to an simple code fragment where only cross clear the terminal?

thx

**Comment by TimonPost** [comment]
Yes, im currently working on that. I'll will make feature flags to allow you to only have the clearing functionality of crossterm.

If you are on Windows 10 ANSI escape codes are used other windows versions are using winapi. Before you can execute ANSI escape codes on windows you have to execute on winapicall to enable this functionality. 

I can link you some links later due the fact I'm on my phone now.

**Comment by cyberpunkbln** [comment]
Great thx and for the links,later.

Your crate functioned very well, before i dig deep into the native GUI-Development (i'm using the gnu toolchain on windows and this make it something stranger and i search something rust pure guis and the mostly guis are wrapper for c/c++ libs/guis:)) i will test software with an terminal only user interface.

Yeah the feature feature :) sounds very good.

**Comment by TimonPost** [comment]
Nice! 
So here you can find some links I use for clearing the console screen:

[This ](https://github.com/TimonPost/crossterm/blob/master/src/modules/terminal/ansi_terminal.rs#L21)is the line for clearing screen with ANSI escape codes: 

[Here ](https://github.com/TimonPost/crossterm/blob/master/src/common/commands/win_commands.rs#L28)you can find the code responsible for enabling ANSI-codes on windows.

**Comment by TimonPost** [comment]
Yes, that is totally true. You can just use the crate right now and when the feature flags will become available it will be easy to add them.
